apps/src/storage/levelbuilder/DatasetList.jsx

```diff
@@ -1,8 +1,27 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 class DatasetList extends React.Component {
+  static propTypes = {
+    datasets: PropTypes.arrayOf(PropTypes.string).isRequired,
+    liveDatasets: PropTypes.arrayOf(PropTypes.string).isRequired
+  };
   render() {
-    return <h1>Datasets</h1>;
+    return (
+      <div>
+        <h1>Datasets</h1>
+        <ul>
+          {this.props.datasets.map(dataset => (
+            <li key={dataset}>
+              <a href={`/datasets/${dataset}`}>{dataset}</a>
```

Comment:
For accessibility, should this include a title attribute? i.e. "{dataset} dataset" -> "Dogs dataset"

Reply:
(also, is levelbuilder within our accessibility goals?)

Reply:
I don't think that would make this more accessible (header on the page is Datasets and the links are `/datasets/{name}` so I think it is already contextually clear that the list contains the names of datasets).

Reply:
👍